### PR TITLE
Upgrade SockJS to 1.0.3

### PR DESCRIPTION
**Pull request by trestletech**
We should continue to only update on `n.0.3` releases to keep these diffs small.

### CODE REVIEW

**Comment by trestletech** [comment]
I tested that the protocols continue to work, and basic event handling seems sane. I'm not sure how to exhaustively test that the event API is unchanged. We'll just need to manually soak it before release.

File: assets/shiny-server.js [hunk 1]
```diff
@@ -388,7 +388,7 @@
       url = url.replace(/\/n=/, '/o=');
     }
     var conn = new SockJS(url,
-      null,{protocols_whitelist: this._whitelist});
+      null,{transports: this._whitelist, sessionId: 16});
```
Comment (aronatkins) [inline assets/shiny-server.js]: The old session id was 8 characters, yes? Do any REs need changing to allow for more characters?

**Comment by aronatkins** [comment]
:+1: other than the question about session length.

File: assets/shiny-server.js (hunk 1 above)
Comment (trestletech) [inline assets/shiny-server.js]: No, we just use `\w+`

**Pull request merged**